inst/stan/estimate_secondary.stan

```diff
@@ -8,8 +8,8 @@ functions {
 }
 
 data {
-  int t;                             // time of observations
-  int lt;                             // time of observations
+  int t;                             // total time (length of input data)
```

Comment:
I think it would be good to define this consistently across models - also isn't `lt` the length of the input data?
```suggestion
  int t;                             // total modelled time
```